dts/arm/nordic/nrf9160.dtsi

```diff
@@ -28,10 +28,11 @@
 	aliases {
 		flash-controller = &flash_controller;
 		ficr = &ficr;
+		rtc-0 = &rtc0;
```

Comment:
Could we add PWM peripherals as well? 
For reference: https://github.com/barsok/fw-nrfconnect-zephyr/commit/f9385b4b19fca1f0455af079e9e700454390af02

Reply:
Good point. I'll add them in #12531.

Reply:
They are added. Resolving this one.